pkg/registry/generic/etcd/etcd_test.go

```diff
@@ -339,6 +339,53 @@ func TestEtcdUpdate(t *testing.T) {
 
 }
 
+func TestNoOpUpdates(t *testing.T) {
+	server, registry := NewTestGenericEtcdRegistry(t)
```

Comment:
I can, but we don't have such, so this would make this change much bigger (it will take some time to implement it).